src/openforms/conf/base.py

```diff
@@ -596,6 +596,7 @@
     "DEFAULT_THROTTLE_RATES": {"anon": "200/day", "user": "1000/day"},
     "TEST_REQUEST_DEFAULT_FORMAT": "json",
     "DEFAULT_SCHEMA_CLASS": "openforms.api.schema.AutoSchema",
+    "EXCEPTION_HANDLER": "openforms.api.views.handler",
```

Comment:
`handler` really says nothing - and you should really follow the upstream pattern. We're overriding the default DRF exception handler: https://github.com/encode/django-rest-framework/blob/master/rest_framework/views.py#L71

Please use the same function name.